src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -143,35 +146,46 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
     *******************************************************************************/
 
     /**
-     * @notice Registers msg.sender as an operator with the middleware
-     * @param quorumNumbers are the bytes representing the quorum numbers that the operator is registering for
-     * @param socket is the socket of the operator
+     * @notice Registers msg.sender as an operator for one or more quorums. If any quorum reaches its maximum
+     * operator capacity, this method will fail.
+     * @param quorumNumbers is an ordered byte array containing the quorum numbers being registered for
      */
     function registerOperator(
         bytes calldata quorumNumbers,
         string calldata socket
     ) external onlyWhenNotPaused(PAUSED_REGISTER_OPERATOR) {
-        (uint32[] memory numOperatorsPerQuorum, ) = _registerOperator({
-            operator: msg.sender, 
+        bytes32 operatorId = blsPubkeyRegistry.getOperatorId(msg.sender);
```

Comment:
why do we need this idToRegister stuff?

Reply:
oh yeah, that's from an earlier change. ill fix the naming, good call out

Reply:
Done in [23eb1a3](https://github.com/Layr-Labs/eigenlayer-middleware/pull/57/commits/23eb1a3e7cf4f2e4f81af138d307a31355e56cb5)